codalab/lib/interactive_session.py

```diff
@@ -23,15 +24,15 @@ class InteractiveSession:
         3. Mount the dependencies as read-only.
         4. Set working directory to some arbitrary path: `/<session uuid>`
         5. The user will interact with this container and try out different commands. Once satisfied the user will
-           exit the bash session. All the commands the user tried out will be stored at path `/root/.bash_history`
+           exit the bash session. All the commands the user tried out will be stored at path `/usr/sbin/.bash_history`
            in the container.
         6. Copy `.bash_history` out to the host machine.
         7. Open an editor and allow the user to select and edit commands for the official run.
         8. Return the final command.
         9. Stop and remove the interactive session container.
     """
 
-    _BASH_HISTORY_CONTAINER_PATH = "/root/.bash_history"
+    _BASH_HISTORY_CONTAINER_PATH = "/usr/sbin/.bash_history"
```

Comment:
Is there anything special about this particular path?  More specifically, what property do we need to satisfy here...

Reply:
It just has to be on a non-root path. I think `/usr/bin` should work too.